docs/sphinx/reference-core.rst

```diff
@@ -278,6 +278,8 @@ Libobs Objects
    :c:func:`obs_source_get_weak_source()` if you want to retain a
    reference after obs_enum_sources finishes.
 
+   For scripting, use :py:func:`obs_enum_sources`.
```

Comment:
Any reason you went with `:py:` instead of `:c:`?

Reply:
That's under .. function:: void obs_enum_sources(bool (*enum_proc)(void*, obs_source_t*), void *param), which is a C function. If I used :c:, that would link to the same function it's under. I used :py: so it links to the python documentation for the function of the same name. Remember that many of them have different function signatures from their C function equivalents.

Reply:
Right, good point!